rust/src/writer/stats.rs

```diff
@@ -528,7 +698,7 @@ mod tests {
                 ("some_bool", ColumnCountStat::Value(v)) => assert_eq!(100, *v),
                 ("some_string", ColumnCountStat::Value(v)) => assert_eq!(100, *v),
                 ("some_list", ColumnCountStat::Value(v)) => assert_eq!(100, *v),
-                ("some_nested_list", ColumnCountStat::Value(v)) => assert_eq!(0, *v),
+                ("some_nested_list", ColumnCountStat::Value(v)) => assert_eq!(100, *v),
```

Comment:
I believe this test was incorrect, given the null values here:

https://github.com/delta-io/delta-rs/blob/0ee692cf74bed6ce259a7e883c34c3f0fe407569/rust/src/writer/stats.rs#L885